apps/fabric-website/src/pages/Overviews/GetStartedPage/docs/web/GetStartedDevelop.md

```diff
@@ -1,184 +1,56 @@
-
-```bash
-npm install office-ui-fabric-react
-```
-
-If you have npm start command still running, terminate the command by pressing Ctrl+C.
-
-### Step 3: Install all dependencies
-
-All dependencies of the sample project need to be installed before the project can be launched. To install all dependencies, type the following:
-
-```bash
-npm install
-```
-
-### Step 4: Add PrimaryButton component
-
-Open the `button-demo` project folder in Visual Studio Code.
-
-Open the file `src\App.tsx` and replace the existing code inside `return` to only return the `PrimaryButton` component:
-
-```tsx
-import * as React from 'react';
-import './App.css'
-
-export App: React.FC = () => {
-  return (
-    <PrimaryButton onClick={() => alert("You clicked the button!")}>
-      Hello World
-    </PrimaryButton>
-  );
-}
-```
-
-Since `office-ui-fabric-react` is a npm module, we will need to import the button component in the `App` component.
-
-The following import statement at the top of the `src\App.tsx` file just below `import './App.css'`:
+This scaffold uses the [`just`](https://github.com/microsoft/just) build library. It is very flexible and requires no "eject" script to allow for customization on its configurations.
 
-```tsx
-import { PrimaryButton } from 'office-ui-fabric-react';
-```
+## Create React App
 
-Save the `src\App.tsx` file.
+UI Fabric also provides a precanned `create-react-app` version of the above application due to its popularity and community support.
 
-Below is an example of how you can similarly import other components:
+Clone or fork this repo: https://github.com/kenotron/create-react-app-uifabric
 
-```tsx
-import { DefaultButton, DocumentCard, DocumentCardPreview, DocumentCardTitle, DocumentCardActivity } from 'office-ui-fabric-react';
+```shell
+git clone https://github.com/kenotron/create-react-app-uifabric.git my-app
+cd my-app
+yarn
+yarn start
 ```
 
-Open `index.html` file.
-
-Fabric React components provide support for Left-to-Right (LTR) and Right-to-Left (RTL) rendering out of the box.
+## Gatsby.js
 
-You can specify the default direction in your `index.html`. Add the `dir` attribute with the value ltr to the `<html>` element:
-
-```html
-<html lang="en" dir="ltr"></html>
-```
+To create blazing fast React websites and apps using UI Fabric and Gatsby.js, use the following starter kit:
 
-Load Office UI Fabric styles by linking to the Office UI Fabric CDN. Add the following to the `<head>` element:
-
-```html
-<link rel="stylesheet" href="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-core/9.6.1/css/fabric.min.css" />
-```
-
-Save the file.
-
-### Step 5: Preview button
-
-Switch to the console and type the following to preview your app in the browser:
-
-```bash
-npm start
+```shell
+npm install -g gatsby-cli
+gatsby new gatsby-site kenotron/gatsby-starter-uifabric
```

Comment:
This assumes they have gatsby installed

Reply:
Line 41 is installing gatsby-cli, so this should be fine.